master/setup.py

```diff
@@ -453,7 +453,7 @@ def define_plugin_entries(groups):
     'txaio ' + txaio_ver,
     'autobahn ' + autobahn_ver,
     'PyJWT',
-    'distro'
+    'distro;platform_system==="Linux"'
```

Comment:
If this is critical, the alternative would be something like

```
import platform
...
if platform.system() == 'Linux':
    setup_args['install_requires'].append('distro')

```
If you prefer this, I can update the pull request.